test/unit/component/parser/xml/BpmnXmlParser.bic-cloud-design-A_3_0_0.test.ts

```diff
@@ -0,0 +1,813 @@
+/**
```

Comment:
could you rename the file to follow xml parsing test files like in `BpmnXmlParser.bizagi-modeler-2_8_0_8.test.ts` (vendor software version, no name of the miwg test diagram)